apps/aecore/include/trees.hrl

```diff
@@ -1,5 +1,5 @@
 -record(trees, {
-          accounts :: aec_trees:tree()}).
+          accounts :: undefined | aec_accounts:tree()}).
```

Comment:
Do we really need undefined. What happens if we just leave it out? We cannot create undefined accounts!

Reply:
Without undefined, dialyzer spots that the block is created in some points without accounts (because of [default value in record](https://github.com/lucafavatella/epoch/blob/6ccbeaeb42d0fe39cf6923182da447d08653da61/apps/aecore/include/blocks.hrl#L27)). (Somehow these changes enabled dialyzer to spot this so I had to add the undefined here.)

Reply:
Will remove in following PR, that will remove trees from block.